### PR TITLE
Add Custom logging formatter

### PR DESCRIPTION
**Pull request by SeoFernando25**
fixes: #90 

Replaced the default logging timestamp to `std::chrono::steady_clock` 

**Edit**: also moved files from `helpers` folder to `common`

Before:
image

After:
image

### CODE REVIEW

**Comment by ajayyy** [comment]
Maybe we can show both? The human readable ones can be nice

**Comment by SeoFernando25** [comment]
image

image

> Maybe we can show both? The human readable ones can be nice

Would that work? I changed it so only the `file`sink` uses it. `stdout` prints readable format and `log` uses timestamp

**Review by ajayyy — APPROVED** [review]

**Comment by SeoFernando25** [comment]
haven't pushed it yet

**Comment by SeoFernando25** [comment]
I believe that's it

File: src/common/LogFormatter.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include "pch.h"
+#include "spdlog/pattern_formatter.h"
+#include "utils.h"
+
+/**
+ * Custom formatter for spdlog. Same as default but using the getTimeStamp() function
+ */
+class LogFormatter : public spdlog::formatter
```
Comment (jonapap) [inline src/common/LogFormatter.h:9]:
```suggestion
class FileLogFormatter : public spdlog::formatter
```

This is only used for files, right?

**Comment by jonapap** [comment]
Also, do you know how to use git rebase to merge commits together? If so, would you mind merging some of them in this branch? If you don't know how, its fine, you can leave it as is.

File: src/common/LogFormatter.h (hunk 1 above)
Comment (SeoFernando25) [reply src/common/LogFormatter.h:9]: I'm kinda bad with names lol

**Comment by SeoFernando25** [comment]
Is that it? 👀

**Review by jonapap — APPROVED** [review]
Looks good! I think this can be merged for the HotFire.

**Pull request merged**